pkgs/development/web/kore/default.nix

```diff
@@ -1,19 +1,30 @@
-{ stdenv, fetchFromGitHub, openssl }:
+{ stdenv, fetchFromGitHub, openssl, curl, postgresql, yajl }:
 
 stdenv.mkDerivation rec {
   pname = "kore";
-  version = "3.3.1";
+  version = "4.0.1";
 
   src = fetchFromGitHub {
     owner = "jorisvink";
     repo = pname;
-    rev = "${version}-release";
-    sha256 = "0jlvry9p1f7284cscfsg04ngbaq038yx3nz815jcr5s3d2jzps3h";
+    rev = version;
+    sha256 = "0186lih30zps2d4600ikafbgsml269jzpcszdggzzkdw8p628qw9";
   };
 
-  buildInputs = [ openssl ];
+  buildInputs = [ openssl curl postgresql yajl ];
 
-  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "ACME=1"
+    "TASKS=1"
+    "PGSQL=1"
+    "JSONRPC=1"
+    "DEBUG=1"
```

Comment:
You are sure we want to enable DEBUG?

Reply:
`DEBUG` only enables an option, allowing to run applications in debug mode which is desireable for people developing `kore` applications on NixOS.

> `DEBUG=1` (enables use of -d for debug)

However, it could be an idea to make the build flags configurable using the derivation's arguments.

Reply:
See also: <https://github.com/jorisvink/kore#building-kore>